styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  background: #fff;
+  background-image: url(./Images/image_geometry_1.svg);
+  background-repeat: no-repeat;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  margin: 20px 20px 0 0;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #1a2236;
+  margin: 20% 5% 0 5%;
+}
+
+h1 {
+  font-family: "Playfair Display", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.intro {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  margin-top: 10px;
+  max-width: 500px;
+  color: #1a2236;
+}
+
+.collab-button {
+  text-decoration: none;
+  max-width: 500px;
+  width: 100%;
+  height: 50px;
+  margin: 30px 0;
+}
+
+.button {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  width: 100%;
+  height: 100%;
+  background: #1a2236;
+  color: #fff;
+}
+
```

Comment:
[OPTIONAL] 👍🏽 
- I think it will be better for you to add a hover effect on your button. You can simply do that by adding the line of code I have attached below 👇🏽 
```
.button:hover {
   background: #fff;  
  color: #1a2236;
}
```
What do you think about this? Besides, it is one one the project's requirement as well. 🍷 

![hover](https://user-images.githubusercontent.com/99246359/198520633-0bf34fd8-7da6-487e-a2b7-7ae34ab19d2b.PNG)

Reply:
@Favourezeugwa
- Applied changes to the button as per the project's requirement.